src/lib/components/documents/Zoom.svelte

```diff
@@ -29,77 +29,12 @@ It exports a $zoom store that can be passed around to other components.
 
 <script lang="ts">
   import { _ } from "svelte-i18n";
+  import { getDefaultZoom, getZoomLevels } from "@/lib/utils/viewer";
 
   export let mode: ViewerMode;
 
   $: zoomLevels = getZoomLevels(mode);
   $: $zoom = getDefaultZoom(mode);
-
```

Comment:
If we're only using the `getDefaultZoom` and `getZoomLevels` functions in this component, I'd rather keep them here. I know it's easier to test things if they're separate, but if that's the concern, maybe we move them to module context instead. I don't want to create utility modules for things only used in one place, if we can help it.

Reply:
Yeah, totally fair. I thought they were imported by other files, but turns out they aren't! So the whole utils/viewer can be removed and this logic back into the Zoom component